src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs

```diff
@@ -346,7 +346,8 @@ protected bool BuildInternal (string projectOrSolution, string target, string []
 
 				if (nativeCrashDetected) {
 					Console.WriteLine ($"Native crash detected! Running the build for {projectOrSolution} again.");
-					File.Move(processLog, processLog + ".bak");
+					if (attempt == 0)
+						File.Move(processLog, processLog + ".bak");
```

Comment:
If we're doing a new commit anyway, can we fix the formatting here so that we have a space before `(`?